metaflow/plugins/cards/card_modules/base.html

```diff
@@ -18,6 +18,7 @@
 <body>
   <div class="card_app"></div>
   <script>
+    var METAFLOW_RELOAD_TOKEN = "[METAFLOW_RELOAD_TOKEN]"
```

Comment:
total nit, but we use let/const these days to be more explicit on if the value is immutable.